### DISCUSSION TITLE
HOW to ?

### DISCUSSION THREAD

**Discussion by paulkelele** [bcgit/bc-java#1171] [Q&A] [answered]
Hi every one,

I'm trying to use BC version 1.68 to verify the validity of a certificate. I've got the certificate to check, the issuer certificate and the ocsp provier URL...So from now I need to check if it's GOOG, REVOCATED or UNKNOWN...

I think it should be simple.

So What is the procedure to do that ? Where is the documentation pliz ? 

Also I don't find the package org.bouncycastle.jce.provider.test and all others test packages...

**Comment by dghgit** [comment, marked as answer]
You can find the test code for the providers at: https://github.com/bcgit/bc-java/tree/master/prov/src/test
